docs/source/notebook.md

```diff
@@ -332,6 +332,16 @@ to remember are the following:
 For the full list of available shortcuts, click {guilabel}`Help`,
 {guilabel}`Keyboard Shortcuts` in the notebook menus.
 
+## Searching
+
+Jupyter Notebook has an intelligent built-in search plugin for finding text within a
+notebook or other document, which uses the {kbd}`Ctrl-F` shortcut by default.
+
+Your browser's `find` function will give unexpected results because it doesn't have
+access to the full content of a document, but you can still use your browser find
+function from the browser menu if you want, or you can disable the intelligent search
+shortcut using the advanced settings editor.
```

Comment:
Should Advanced Settings Editor be capitalized as in UI? 

```suggestion
shortcut using the Advanced Settings Editor.
```

![Screenshot 2023-06-09 at 4 13 45 PM](https://github.com/jupyter/notebook/assets/26686070/cfab848a-fa49-478e-be07-d226b7b9c17f)

This also reminds me that we use JSON Settings Editor and Advanced Settings Editor terms interchangeably in different spots but this is out of scope of this PR.